### ISSUE TITLE
Error during activation of LSP-intelephense on Windows 11 in ST4

### ISSUE THREAD

**Issue by frankzimper** [sublimelsp/LSP-intelephense#93]
I can install the package without a problem in Sublime Text 4, but when it gets activated, it fails due to not being able to find the node executable. See attached error log. This seems to be a Windows specific problem, at least I had no issue on Linux so far.

I was able to solve this by installing [lsp_utils](https://github.com/sublimelsp/lsp_utils) before installing LSP-intelephense. This should either be fixed or prominently documented.

[2023-10-27T19_23_35_264Z-debug-0.log](https://github.com/sublimelsp/LSP-intelephense/files/13193033/2023-10-27T19_23_35_264Z-debug-0.log)

**Comment by rchl** [comment]
When you got the issue have you tried just restarting ST and seeing if it works then?

I don't quite see from the log why it would fail.

The log shows that `node` is already installed (it's also required to run `npm` but then later some command fails with:

```
302 error command C:\Windows\system32\cmd.exe /d /s /c node scripts/postinstall
303 error Der Befehl "node" ist entweder falsch geschrieben oder
303 error konnte nicht gefunden werden.
304 verbose exit 1
```

indicating that `node` is not in the PATH.

**Comment by rchl** [comment]
> I was able to solve this by installing [lsp_utils](https://github.com/sublimelsp/lsp_utils) before installing LSP-intelephense. This should be either be fixed or prominently documented.

How did you install `lsp_utils` before `LSP-intelephense`? `lsp_utils` is a dependency - it's not possible to install it separately/manually.

**Comment by rchl** [comment]
Found the issue: https://github.com/sublimelsp/lsp_utils/pull/111

**Comment by frankzimper** [comment]
> When you got the issue have you tried just restarting ST and seeing if it works then?
Yes. Didn't help.

 
> The log shows that `node` is already installed (it's also required to run `npm` but then later some command fails with:
> How did you install `lsp_utils` before `LSP-intelephense`? `lsp_utils` is a dependency - it's not possible to install it separately/manually.

There was some hint by a very nice gentleman which helped me to find this solution. Let me check...
Ah! Here we are: https://github.com/sublimelsp/LSP-intelephense/issues/72#issuecomment-1001581724 :grin:

**Comment by predragnikolic** [comment]
I cannot reproduce this on Windows.

Recently `lsp_utils` changed the local node version from Node 16 to Node 18.

After [this](https://github.com/sublimelsp/lsp_utils/releases/tag/v3.0.0) recent update,
because `lsp_utils` is trying to find the Node 18 path.
I got an (error)popup saying the Node was not found.
and I had a button "Download Node.js", which when pressed installed the new Node 18 version, and delete the old Node 16 version.

---

Do you see the "Download Node.js" button in the popup and have you pressed it?

**Comment by frankzimper** [comment]
> Do you see the "Download Node.js" button in the popup and have you pressed it?

Yes, that popped up every time after a new install of the LSP-intelephense package. I was able to reduce the issue multiple times yesterday with my Sublime Text installation on the Win11 VM. ST4 and the PHP stuff was originally installed on this VM on October 14th. So this predated that recent update you mentioned. 

This morning I uninstalled ST4 and removed everything I was able to find related to ST4. Afterwards I reinstalled Sublime Text, LSP and LSP-intelephense and had **no** issue. So you might be right that this update fixed the issue also.

I'll try to update ST packages on my work machine on Monday. That was where I encountered the issue originally. At home I use Linux so never had the issue.
